clients/src/main/java/org/apache/kafka/clients/admin/ConsumerGroupDescription.java

```diff
@@ -19,53 +19,88 @@
 
 import org.apache.kafka.common.utils.Utils;
 
-import java.util.List;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.Objects;
 
 /**
  * A detailed description of a single consumer group in the cluster.
  */
 public class ConsumerGroupDescription {
+    private final static String PREPARING_REBALANCE = "PreparingRebalance";
+    private final static String COMPLETING_REBALANCE = "CompletingRebalance";
+    private final static String DEAD = "Dead";
+    private final static String STABLE = "Stable";
 
     private final String groupId;
     private final boolean isSimpleConsumerGroup;
-    private final List<MemberDescription> members;
+    private final Collection<MemberDescription> members;
     private final String partitionAssignor;
+    private final String state;
```

Comment:
ConsumerGroupCommand should be updated to use the new KafkaAdminClient (as part of KAFKA-6884). Existing implementation expects "coordinator" and "assignmentStrategy" too in the return value of describeConsumerGroup(), see
 https://github.com/apache/kafka/blob/c3921d489f4da80aad6f387158c33ec2e4bca52d/core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala#L604 

Can you please add these fields too?

Reply:
That is a good point.  "coordinator" should be returned here.  "partitionAssignor" is already available.